test/helpers.bash

```diff
@@ -430,11 +430,16 @@ function cleanup_test() {
         cat "$CRIO_LOG"
         echo "# --- --- ---"
     fi
-    cleanup_ctrs
-    cleanup_pods
-    stop_crio
-    cleanup_lvm
-    cleanup_testdir
+
+    if [ -z "$TEST_KEEP_ON_FAILURE" ] || [ "${BATS_TEST_COMPLETED:-}" = "1" ]; then
```

Comment:
should we do != 0 in case it exits with a different code?

Reply:
Do you mean the failure condition test `"$BATS_TEST_COMPLETED" = "1"` bit ?

If you do, then that specific bit I dug that out of the [BATS documentation](https://bats-core.readthedocs.io/en/stable/faq.html#how-can-i-check-if-a-test-failed-succeeded-during-teardown). It says the BATS_TEST_COMPLETED variable will be set to 1 if the test passed (and I believe also in the case the test is skipped, at least that's what my test runs indicate), otherwise it is left unset.

Reply:
Ah I see, could we have a link here to describe that?

Reply:
Here is the BATS docs bit describing how test completion is reflected by `BATS_TEST_COMPLETED`: https://bats-core.readthedocs.io/en/stable/faq.html#how-can-i-check-if-a-test-failed-succeeded-during-teardown

If you consider it necessary, I can add a reference to that in the commit message.